cc-check.go

```diff
@@ -189,7 +196,16 @@ func checkKernelModules(modules map[string]kernelModule) error {
 			if value == expected {
 				ccLog.Infof("Kernel module %q parameter %q has correct value", details.desc, param)
 			} else {
-				return fmt.Errorf("kernel module %q parameter %q has value %q (expected %q)", details.desc, param, value, expected)
+				msg := fmt.Sprintf("kernel module %q parameter %q has value %q (expected %q)", details.desc, param, value, expected)
+
+				// this option is not required when
+				// already running under a hypervisor.
+				if param == "unrestricted_guest" && onVMM {
```

Comment:
Ideally, rather than special case this one parameter, it would be nicer if the requirement were coded into the data structures - but, this is a test program, and we only have the single case to check, so not a blocker :-)